backend/src/models/answer.js

```diff
@@ -10,3 +10,16 @@ exports.getAnswers = function (questionID, pageNumber) {
         )
     })
 }
+
+exports.postAnswer = function (questionID, {body}) {
+    return new Promise((resolve, reject) => {
+        const columns = ['userID', 'questionID', 'body']
+        const placeholders = columns.map(_ => '?').join()
+        const query = `INSERT INTO answer (${columns}) VALUES (${placeholders})`
+        db.run(
+            query,
+            [0, questionID, body], // TODO user id is a placeholder obviously, but it can't be null so...
+            function (err) { err ? reject(err) : resolve(err) }
```

Comment:
Resolving err?

Reply:
Actually, I want it to resolve getAnswers so it returns the updated list of answers...
This doesn't seem to work though `resolve(exports.getAnswers(questionID, 1))

Reply:
hmm yeah idk what's going on; i just get `TypeError: Cannot read property 'then' of undefined`